src/stratis_cli/_actions/_stratisd_version.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2020 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Check version of stratisd
+"""
+# isort: THIRDPARTY
+from semantic_version import SimpleSpec, Version
+
+from .._errors import StratisCliStratisdVersionError
+from ._connection import get_object
+from ._constants import TOP_OBJECT
+
+
+def check_stratisd_version():
+    """
+    Checks that the version of stratisd that is running is compatible with
+    this version of the CLI
+
+    :raises StratisCliStratisdVersionError
+    """
+    from ._constants import MAXIMUM_STRATISD_VERSION, MINIMUM_STRATISD_VERSION
+    from ._data import Manager
+
+    version_spec = SimpleSpec(
+        ">=%s-,<%s-" % (MINIMUM_STRATISD_VERSION, MAXIMUM_STRATISD_VERSION)
```

Comment:
I believe this spec string could be moved into ._data.py or ._constants.py

Reply:
> I believe this spec string could be moved into _data.py or _constants.py
> The "-" allows for the inclusion of pre-release versions. This may not be necessary at the moment.

We decided in stand up to ignore the possibility of pre-release versions for not. It is probably best to leave the spec string right where it is, since it is so specific to this particular library.